packages/lua-lpeg/build.sh

```diff
@@ -8,6 +8,10 @@ TERMUX_PKG_SRCURL=https://www.inf.puc-rio.br/~roberto/lpeg/lpeg-$TERMUX_PKG_VERS
 TERMUX_PKG_SHA256=4b155d67d2246c1ffa7ad7bc466c1ea899bbc40fef0257cc9c03cecbaed4352a
 TERMUX_PKG_BUILD_DEPENDS="liblua51, liblua52, liblua53, liblua54"
 TERMUX_PKG_DEPENDS="liblua54"
+# thanks Maxython for the tip
+# this variable has gone unused since being introduced in September 2023
```

Comment:
Credit and history is nice, but I think it is better to put it in commit message rather than in build recipe

Reply:
Yeah you're probably right, I was just mildly irritated that there is absolutely no documentation other than the line in `buildorder.py` about this control variable.
https://github.com/termux/termux-packages/blob/67f175612833f64bd9f2d05325c1c23eaedbe3a2/scripts/buildorder.py#L117

I'll do a force push.